boards/hail/src/main.rs

```diff
@@ -154,6 +160,9 @@ unsafe fn set_pin_primary_functions() {
     PB[15].configure(None); //... D1
 }
 
+/// Reset Handler.
+///
+/// This symbol is loaded into vector table by the SAM4L chip crate.
```

Comment:
Maybe explain what `reset_handler` is instead of saying it's in the vector table (which I have a feeling is fairly meaningless to non-embedded people)